package FixedAudioModel;
option java_package="com.thinkit.protobuf.msg";
option java_outer_classname="FixedAudioModel";

//任务头
message TaskHeader {
	required fixed32 taskLength = 1;
}

//结果头(定长10字节)
message ResultHeader{
	required fixed32 resultLength = 2;//结果长度
}
	
//模板信息
message ModelInfo{	
      enum StatusCode {
            FALURE = 0;
            SUCCESS = 1;
	}
	required string taskid = 1;//任务id
	required StatusCode status = 2;//状态码
	required bytes modelUrl = 3;//模板url
}

// 模板下载结果
message LoadResult
{
        enum StatusCode {
            FALURE = 0;
            SUCCESS = 1;
	}
	required string taskid = 1;//任务id
	required StatusCode status =2;//下载状态码
}